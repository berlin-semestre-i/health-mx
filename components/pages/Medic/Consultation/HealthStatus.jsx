import React from 'react'
import { Dropdown } from 'semantic-ui-react'
import Card from '../../../shared/Card'

const healthStatusOptions = [{
  text: 'Regular', value: 0,
}, {
  text: 'CrÃ­tico', value: 1,
}]

const HealthStatus = ({ healthStatus, onChange }) => (
  <React.Fragment>
    <Card header="Estado de Salud">
      <Dropdown
        name="healthStatus"
        value={healthStatus}
        placeholder="Seleccionar Estado"
        fluid selection
        options={healthStatusOptions}
        onChange={onChange}
      />
    </Card>
  </React.Fragment>
)

export default HealthStatus