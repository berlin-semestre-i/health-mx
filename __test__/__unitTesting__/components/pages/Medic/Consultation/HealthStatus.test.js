import React from 'react'
import { shallow } from 'enzyme'
import HealthStatus from '../../../../../../components/pages/Medic/Consultation/HealthStatus'

describe('HealthStatus', () => {
  it('renders', () => {
    const component = shallow(<HealthStatus />)
    expect(component).toMatchSnapshot()
  })
})