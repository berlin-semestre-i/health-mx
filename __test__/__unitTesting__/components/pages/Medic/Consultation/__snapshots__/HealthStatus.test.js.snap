// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HealthStatus renders 1`] = `
<Fragment>
  <CustomCard
    header="Estado de Salud"
  >
    <Dropdown
      additionLabel="Add "
      additionPosition="top"
      closeOnBlur={true}
      deburr={false}
      fluid={true}
      icon="dropdown"
      minCharacters={1}
      name="healthStatus"
      noResultsMessage="No results found."
      openOnFocus={true}
      options={
        Array [
          Object {
            "text": "Regular",
            "value": 0,
          },
          Object {
            "text": "CrÃ­tico",
            "value": 1,
          },
        ]
      }
      placeholder="Seleccionar Estado"
      renderLabel={[Function]}
      searchInput="text"
      selectOnBlur={true}
      selectOnNavigation={true}
      selection={true}
      wrapSelection={true}
    />
  </CustomCard>
</Fragment>
`;
